plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/models/my_new_project/issue_1608.sql.after

```diff
@@ -8,7 +8,7 @@ final as
 (
     select
         col_name,
-        {{- echo('col_name') -}}        as col_name2
+        {{- echo('col_name') -}} as col_name2
```

Comment:
The new test results are clearly better -- the old way, we were checking that L003 added "indentation" in the middle of a line. 🤯🤣